.travis.yml

```diff
@@ -3,7 +3,8 @@
 language: ruby
 cache: bundler
 script:
-  - bundle exec jekyll algolia --verbose
```

Comment:
I need this to update the search index though, right?

Reply:
@jpasholk I don't think so, I made some changes to the repo so the tests would pass. If you pull down this branch, install the gems locally and run the script it should come back with a pass. Let me know your results